docs/DeveloperGuide.adoc

```diff
@@ -767,7 +767,154 @@ Use case ends.
 +
 Use case resumes at step 2.
 
-{More to be added}
+[discrete]
+=== Use case: UC2 -  Export Contacts
+
+*MSS*
+
+1.  User selects contacts to be exported
+2.  User requests for creating file containing selected contacts
+3.  AddressBook creates file with specified user
+4.  User specifies the location where data file needs to be stored
+5.  AddressBook saves the data file in proper format and indicates success
+6.  User sends the data file to the required recipient
++
+Use case ends.
+
+*Extensions*
+
+[none]
+* 1a. No contacts selected
++
+Use case ends.
+
+* 3a. Address Book unable to create file with mentioned contacts
++
+[none]
+** 3a1. AddressBook shows an error message.
++
+Use case resumes at step 1.
+* *a. At any time user chooses to cancel exporting contacts
++
+[none]
+** *a1. AddressBook shows an error message.
++
+Use case ends.
+
+
+[discrete]
+=== Use case: UC3 - Import Contacts
+
+*MSS*
+
+1.  User receives the data file containing contacts
+2.  User requests to import contacts
+3.  Address Book requests for file location
+4.  User specifies location where the data file is found
+5.  AddressBook reads the file and merges new contacts it into existing file
++
+Use case ends.
+
+*Extensions*
+
+[none]
+* 4a. No file selected
++
+Use case ends.
+
+* 5a. Address Book finds duplicate contacts
++
+[none]
+** 5a1. AddressBook shows an error message and doesn't merge the duplicate ones
++
+Use case resumes at step 1.
```

Comment:
If there is any duplicate contact, I thought we would just ignore and continue with the merging, instead of immediately ending the merging?